src/CollegeStudent.java

```diff
@@ -24,6 +24,11 @@ public void doAssignment() {
 		}
 	}
 	
+	public void doGroupAssignment(CollegeStudent partner) {
+		int timeSpent = (int) (Math.random()*60+15);
```

Comment:
Good use of the cast back to int on the double to prevent a type mismatch error.